.travis.yml

```diff
@@ -160,3 +160,20 @@ jobs:
       name: 'macOS 10.10  [GOAL: deploy] [no functional tests]'
       env: >-
         FILE_ENV="./ci/test/00_setup_env_mac.sh"
+
+    - stage: test
+      name: 'macOS 10.14 native'
+      os: osx
+      # Use the most recent version:
+      # Xcode 11, macOS 10.14, JDK 12.0.1
+      # https://docs.travis-ci.com/user/reference/osx/#OS-X-Version
+      osx_image: xcode11
+      language:
+        - ruby
+        - python
+      python:
+        - "3.5.6"
+      rvm:
+        - 2.6.0
```

Comment:
This config doesn't boot (even after waiting an hour)

Maybe you need to install python via `brew upgrade python`?

Reply:
Trying with `pyenv install 3.5.6`

Reply:
That doesn't seem to boot either.

Reply:
In def61133e18a4487af1c9406ffa3f6bf19b702fd I removed both of these and it fails with:

https://travis-ci.org/bitcoin/bitcoin/jobs/596568166

Reply:
```
Warning: automake 1.16.1_1 is already installed and up-to-date

To reinstall 1.16.1_1, run `brew reinstall automake`

Warning: berkeley-db@4 4.8.30 is already installed and up-to-date

To reinstall 4.8.30, run `brew reinstall berkeley-db@4`

Warning: libtool 2.4.6_1 is already installed and up-to-date

To reinstall 2.4.6_1, run `brew reinstall libtool`

Warning: boost 1.71.0 is already installed and up-to-date

To reinstall 1.71.0, run `brew reinstall boost`

Warning: miniupnpc 2.1 is already installed and up-to-date

To reinstall 2.1, run `brew reinstall miniupnpc`

Warning: openssl 1.0.2t is already installed and up-to-date

To reinstall 1.0.2t, run `brew reinstall openssl`

Warning: pkg-config 0.29.2 is already installed and up-to-date

To reinstall 0.29.2, run `brew reinstall pkg-config`

Warning: qt 5.13.1 is already installed and up-to-date

To reinstall 5.13.1, run `brew reinstall qt`

Warning: qrencode 4.0.2 is already installed and up-to-date

To reinstall 4.0.2, run `brew reinstall qrencode`

Warning: librsvg 2.46.1 is already installed and up-to-date

To reinstall 2.46.1, run `brew reinstall librsvg`

Warning: pyenv 1.2.13_1 is already installed and up-to-date

To reinstall 1.2.13_1, run `brew reinstall pyenv`

Error: protobuf 3.9.1_1 is already installed

To upgrade to 3.10.0, run `brew upgrade protobuf`.

The command "brew install automake berkeley-db4 libtool boost miniupnpc openssl pkg-config protobuf qt qrencode librsvg pyenv" failed. Retrying, 3 of 3.

Reply:
Oh boy, Homebrew is a pain. I lifted some stuff from:
https://discourse.brew.sh/t/best-practice-for-homebrew-on-travis-brew-update-is-5min-to-build-time/5215/11 and https://github.com/vadi2/Mudlet/blob/857d1ae29fe6e55f1507399200c45db318b28ae6/CI/travis.osx.install.sh#L11-L35

Reply:
@vadi2 @keneanung let me know if you prefer not to be mentioned as co-authored-by in the commit.

Reply:
(assuming we ever get this to work :-)

Reply:
It's fine! Let us know if we can help you out with this.

Reply:
I'm good with this as well. As Vadim said, let us know if we can be of help.

Reply:
Thanks! I could use some help with _update 7_ below.

Reply:
Hm. I'm unsure whether we had this error... And since we don't use this deep caching anymore, we don't have any further experience in this dark corner of homebrew :ghost: